### ISSUE TITLE
Basic IP routing

### ISSUE THREAD

**Issue by mfld-fr** [mfld-fr/elks#12]
Add basic routing to `ktcp` networking process.

**Comment by mfld-fr** [comment]
@georgp24 proposed to work on that item ?

**Comment by georgp24** [comment]
Yes, I did offer that. This includes the serial/ethernet switch issue #10. This should also switch between a loopback driver and maybe in the future an RTL and/or Intel E1000 network driver.

I will use the following configuration files for that:
/etc/network/interfaces
/etc/sysconfig/network
/etc/sysconfig/static-routes

So the ethernet driver should just offer the init(), ioctl(), select(), read() and write() functions and be linked with ktcp just as slip.c. Ktcp then has to read the interfaces file and decide which drivers to init() and query for data or write to.

However, first I will try to get the current elks-master to work just as well as the release 0.2.0.

**Comment by mfld-fr** [comment]
Nothing here to recover
